.github/workflows/container-image.yml

```diff
@@ -18,3 +18,20 @@ jobs:
       run: make build
     - name: Test the image
       run: make test
+
+  push:
+
+    # verify it passes build and test first
+    needs: build
+
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: log in to quay.io
+        run: podman login -u "${{ secrets.QUAY_USER }}" -p "${{ secrets.QUAY_PASS }}" quay.io
+      - name: push server image
+        run: make push-server
+      - name: push client image
+        run: make push-client
```

Comment:
It's no biggie but would it make more sense to just push everything in one step like: `make push-server push-client`?

Reply:
dunno. there is no synergy between the two, right? so this way, we'll see more separate steps in the logs here. 
If you want to club them, I suggest to add a make target  "push" just like the "build" target... ;-)